local-example.env

```diff
@@ -1,12 +1,8 @@
 # For proxy JWT (required)
-AUTH_COOKIE_NAME=
-AUTH_TOKEN_SECRET=
-
-AUTH_MANAGEMENT_API=
-
-AUTH_SITE_ONE_LEVEL=
-AUTH_SITE_ONE=
-AUTH_SITE_TWO_LEVEL=
-AUTH_SITE_TWO=
-AUTH_SITE_THREE_LEVEL=
-AUTH_SITE_THREE=
+COOKIE_NAME=
+# base64 encoded string
+TOKEN_SECRET=
+# Comma separated values, values are in format {level}:{url}
+# For example: "one:server1,two:server2,three:server3"
+SITES=
+MANAGEMENT_API=
```

Comment:
Can you give an example and/or a comment to indicate whether it's a full URL or just a host name?